newIDE/app/src/UI/Theme/DarkTheme/EventsSheet.css

```diff
@@ -26,7 +26,11 @@
  */
 .gd-events-sheet-dark-theme .rst__moveHandle {
   background: #494949 !important;
-  border: 1px solid #494949 !important;
+  border: none!important;
```

Comment:
See my comments in the other PR, we can remove the important if we remove the extra rules in the common css file used by EventsSheet.